README.md

```diff
@@ -16,7 +16,11 @@ Since reader, writer and shared mostly need the same commands, the main Makefile
 
 - `run-cleanup`, `run-tests`, `run-travis`: runs the corresponding command in all three modules
 - `build-dev`: build the dev images for reader and writer (shared has no dev image since it's included in both the reader and the writer)
-- `run-prod`: see below
+- `run-prod`: starts a productive environment. See below for details.
```

Comment:
Maybe change this to
```
* make run-prod [...]
* make run-prod-verbose [...]
[...]
```
When you read the hole file, it is clear, that this are make targets. But when I first scanned the document, it was not clear to me.

It would be nice, if the command, that I have to type into the console could be found entirely in the README.

Reply:
Sure, will do.